snuba/datasets/outcomes_processor.py

```diff
@@ -14,13 +14,21 @@
     _unicodify,
 )
 
+from snuba.utils.metrics.wrapper import MetricsWrapper
+
+metrics = MetricsWrapper(environment.metrics, "outcomes.processor")
+
 
 class OutcomesProcessor(MessageProcessor):
     def process_message(
         self, value: Mapping[str, Any], metadata: KafkaMessageMetadata
     ) -> Optional[ProcessedMessage]:
         assert isinstance(value, dict)
         v_uuid = value.get("event_id")
+
+        if "category" not in value:
+            metrics.increment("missing_category")
```

Comment:
I might add org_id as a tag here. If this does spike up, we'll have a little bit more data to help debug.

Reply:
👍🏼 went ahead and did so!

Reply:
Sorry, please do not add the org id as a tag. org-id and project-id are high cardinality, so we should not add them as tags. (cost).

Reply:
My bad, I thought org id would be low cardinality enough to avoid much cost. The change to split quantity and category is good though.

Reply:
got it -- sounds good.